jsnlog/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for JSNLog 2.17.3
+// Type definitions for JSNLog 2.17.4
```

Comment:
Could you limit this to 2.17? We only hold onto the major.minor version.

If you wouldn't mind adding a `tslint.json`, that will help catch that here. It's as simple as adding a file [like this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/35ce1629a7e5ffba69a582126cd70744b493b71a/table/tslint.json).